src/main/java/duke/FileHandling.java

```diff
@@ -0,0 +1,81 @@
+package duke;
+
+import java.io.File;  // Import the File class
+import java.io.IOException;  // Import the IOException class to handle errors
+import java.io.FileWriter;
+import java.io.FileNotFoundException;  // Import this class to handle errors
+import java.util.Scanner; // Import the Scanner class to read text files
+public class FileHandling {
+    public static void saveContents() {
+        try {
+            File fileObj = new File("tasklist.txt");
+            if (fileObj.createNewFile()) {
+                System.out.println("File created: " + fileObj.getName());
+            } else {
+                System.out.println("File already exists.");
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+        }
+
+        try {
+            FileWriter myWriter = new FileWriter("tasklist.txt");
+            for (Task task : Echo.tasks) {
+                if (task.getSymbol() == "T") {
+                    myWriter.write(task.getSymbol() + "|" + task.getStatusIcon() + "|" + task.description + '\n');
+                } else if (task.getSymbol() == "D") {
+                    Deadline obj = (Deadline) task;
+                    myWriter.write(task.getSymbol() + "|" + task.getStatusIcon() + "|" + task.description + "|" + obj.by + '\n');
+                } else if (task.getSymbol() == "E") {
+                    Event obj = (Event) task;
+                    myWriter.write(task.getSymbol() + "|" + task.getStatusIcon() + "|" + task.description + "|" + obj.start + "|" + obj.end + '\n');
```

Comment:
Coding standard: Line length should be no longer than 120 chars. Use line wrapping to split up long lines like these.